### PR TITLE
New

### PR DESCRIPTION
**Pull request by MaDina47**
Homework 5

### CODE REVIEW

File: src/test/java/Pages/Commands.java [hunk 1]
```diff
@@ -41,4 +59,62 @@ public boolean isElementDisplayed(By locator) {
...
+    // custom method to get all window-handles
+    public Set<String> getAllWindowHandles() {
+        return MyDriver.getDriver().getWindowHandles();
+    }
+
+    // custom method to select date from Calendar
+    public void selectDateInCalendar(By locator, String userDate) {
+        List<WebElement> allDates = MyDriver.getDriver().findElements(locator);
+        for (WebElement dateElement : allDates) {
+            String dateValue = dateElement.getAttribute("data-day");
```
Comment (ahsansiddiki) [inline src/test/java/Pages/Commands.java:97]: Commands page should not be hard coded like **"data-day"**
Rather add 1 more parameter for this and pass the value from page object.

File: src/test/java/Pages/Commands.java [hunk 2]
```diff
@@ -41,4 +59,62 @@ public boolean isElementDisplayed(By locator) {
...
+            if(suggestion.getText().equalsIgnoreCase(userSuggestion)) {
+                suggestion.click();
+                break;
+            }
+        }
+    }
+
+    public void scrollByPixel(){
+        JavascriptExecutor js = (JavascriptExecutor) MyDriver.getDriver();
+        js.executeScript("scrollBy(0,900)");
```
Comment (ahsansiddiki) [inline src/test/java/Pages/Commands.java:117]: You can make this xValue and yValue parameterized.

**Review by ahsansiddiki — APPROVED** [review]
Looks like you missing Cucumber implementations.

**Comment by MaDina47** [comment]
Hi Ahsan! 
Thank you for your feedback! I didn’t implement cucumber because hw5 was before it.

**Pull request merged**